tests/agent/test_agent.py

```diff
@@ -113,13 +112,13 @@ def local_mercury():
 
 
 @pytest.fixture
-def org_id_env_var():
-    return os.environ.get("GX_CLOUD_ORGANIZATION_ID")
+def org_id() -> str:
+    return "3ee9791c-4ea6-479d-9b05-98217e70d341"
 
 
 @pytest.fixture
-def token_env_var():
-    return os.environ.get("GX_CLOUD_ACCESS_TOKEN")
+def token() -> str:
+    return "access_token"
```

Comment:
This is the meat of the change

Reply:
These old fixtures were expressively named, can we make these similar expressive? `dummy_token`, `dummy_org_id`.

Or `random_token`/`random_org_id` (and actually make them random).

Reply:
Good call - I was meaning to keep this PR small but it makes sense to keep all these changes together.

Reply:
Updated